composer.json

```diff
@@ -106,7 +106,8 @@
         "php-http/httplug":               "^2.0",
         "php-http/client-common":         "^2.0",
         "php-http/message-factory":       "^1.0",
-        "php-http/discovery":             "^1.0"
+        "php-http/discovery":             "^1.0",
+        "twig/twig":                      "^1.35.0|^2.4.4|^3.0"
```

Comment:
```suggestion
        "twig/twig":                      "^1.35|^2.4.4|^3.0"
```

Reply:
actually removed it from require and kept the conflicting rule